tools/ZAPD/lib/libgfxd/meson.build

```diff
@@ -0,0 +1,9 @@
+#project('gfxd', 'c')
+
+gfxd = 'gfxd.c'
+uc_srcs = ['uc_f3d.c', 'uc_f3db.c', 'uc_f3dex.c', 'uc_f3dexb.c', 'uc_f3dex2.c']
+
+# todo, add project cflags to replicate O2, (CFLAGS = -Wall -O2 -g)
```

Comment:
-Wall is the default -- the default project warning_level is 1, that includes -Wall.

You can handle -g with the builtin project option debug=true, -O2 by the builtin project option optimization=2.